UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/DataHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.Helper
+{
+    public class DataHelper
+    {
+        public ProductVersionModel ProductVersionmodel { get; set; }
+
+        public ProductVersionModel GetProductVersionModelData(string productName, int? editionNumber, int? updateNumber)
+        {
+            ProductVersionmodel = new ProductVersionModel()
+            {
+                ProductName = productName,
+                EditionNumber = editionNumber,
+                UpdateNumber = updateNumber
+
+
+
+            };
```

Comment:
```suggestion
                ProductName = productName,
                EditionNumber = editionNumber,
                UpdateNumber = updateNumber
            };
```

Reply:
I have done the changes as suggested.
Thanks Greg.